src/components/InternalDropdown/pageObject/index.js

```diff
@@ -36,6 +36,15 @@ class PageInternalDropdown {
             .moveTo();
     }
 
+    /**
+     * It moves the pointer over the menu item.
+     * @method
+     */
+    hover() {
+        const itemElement = $(`${this.rootElement} > div[role="option"]`);
+        itemElement.moveTo();
+    }
```

Comment:
this method is part of the Option component, look at other implementations where we use it like:
```
...
const option2 = dropdown.getOption(2);
option2.hover()
...
```